plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/HiveWriteUtils.java

```diff
@@ -240,7 +240,12 @@ public static Optional<Boolean> directoryExists(TrinoFileSystem fileSystem, Loca
 
     public static boolean isFileCreatedByQuery(String fileName, String queryId)
     {
-        return fileName.startsWith(queryId) || fileName.endsWith(queryId);
+        return fileName.startsWith(queryId)
```

Comment:
we are using the [logic](https://github.com/trinodb/trino/blob/6ff430abf9742f36ac94eec5c63ebd979a170705/plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveWriterFactory.java#L427-L429) to build fileName/Path, and expect the fileNames always startsWith or endsWith queryId, while the `HiveWriteFactory#getFileExtension` possible returns extension, it seems a bug previously